src/Corporation/Actions.ts

```diff
@@ -628,3 +631,37 @@ export function SetProductMarketTA1(product: Product, on: boolean): void {
 export function SetProductMarketTA2(product: Product, on: boolean): void {
   product.marketTa2 = on;
 }
+
+export function Bribe(
```

Comment:
Functions should be lowerCamelCase. I get that this will conflict with `bribe` as a function used elsewhere, but IMO that is a feature, because having two functions distinguished only by case is also a bit of a smell.

I don't have enough context yet to suggest how these should be named differently, but clearly they *should*, since one calls the other. Edit: I might call the other function `uiBribe` or something along those lines, since its responsibilities lie with pulling and verifying inputs from the UI components. Maybe you can think of better.